### DISCUSSION TITLE
CLI工具或者使用文档

### DISCUSSION THREAD

**Discussion by alephpi** [RapidAI/RapidOrientation#22] [Q&A] [answered]
您好，从paddleocr的issue底下发现你们的工作，说实话被paddleocr的屎山代码搞得没脾气。感觉你们这个轻量版的不错，请问版面重建有实例代码吗？我看readme的流程图很清楚，要是有相关代码就更方便了。

**Comment by SWHL** [comment, marked as answer]
版面重建部分暂时还没有相关代码的。欢迎一起来完善
